unison-src/transcripts/add-run.md

```diff
@@ -0,0 +1,59 @@
+# add.run
+
+```ucm:hide
+.> builtins.merge
+```
+
+## happy path
+```unison
+even : Nat -> Boolean
+even x = if x == 0 then true else odd (drop x 1)
+
+odd : Nat -> Boolean
+odd x = if x == 0 then false else even (drop x 1)
+
+is2even : 'Boolean
+is2even = '(even 2)
+```
+
+```ucm:error
+.> add.run foo
+```
+
+```ucm
+.> run is2even
+```
+
+```ucm:error
+.> add.run is2even
+```
+
+```ucm
+.> add.run foo.bar.baz
+```
+
+```ucm
+.> view foo.bar.baz
+```
+
+## It continues to work if a dependency is changed
```

Comment:
I'm not quite sure what this is testing or asserting, is it saying that `add.run` will still use the old unisonfile even if it changes, or is it asserting the opposite?

Maybe it'd be helpful to add something like this:

    ```ucm:hide
    .> builtins.mergeio
    ```

    ```unison
    txt = "initial"

    main : '{IO, Exception} Text
    main _ = txt
    ```

    ```ucm:hide
    .> run main
    ```

    ```unison
    txt = "changed"

    main : '{IO, Exception} Text
    main _ = txt
    ```

    ```ucm
    .> add.run result
    .> view result
    ```